Channels/Production/release_notes.md

```diff
@@ -1,45 +1,28 @@
-## April 24th, 2019 Production Release ([v1.1.0837.02](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0837.02))
+## June 18th, 2019 Production Release ([v1.1.0883.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0883.01))
```

Comment:
@Sirisha-Kommushetty, please double-check the release notes. They're copied from the Insider release notes that you reviewed a couple of weeks ago, then I updated the first 2 lines.

Reply:
Look good to me.